src/go/rdctl/cmd/listSettings.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+*/
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// listSettingsCmd represents the listSettings command
```

Comment:
Not sure this is worth a whole file, but 🤷

Reply:
The small files are the result of running `cobra generate`, but I don't mind them.